mxnet/buildspec.yml

```diff
@@ -41,126 +41,126 @@ context:
+    os_version: &OS_VERSION ubuntu20.04
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    target: sagemaker
+    context:
+      <<: *TRAINING_CONTEXT
+#  BuildEC2MXNetCPUInferencePy3DockerImage:
+#    <<: *INFERENCE_REPOSITORY
+#    build: &MXNET_CPU_INFERENCE_PY3 false
+#    image_size_baseline: 2202
+#    device_type: &DEVICE_TYPE cpu
+#    python_version: &DOCKER_PYTHON_VERSION py3
+#    tag_python_version: &TAG_PYTHON_VERSION py38
+#    os_version: &OS_VERSION ubuntu20.04
+#    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
+#    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+#    target: ec2
+#    context:
+#      <<: *INFERENCE_CONTEXT
+#  BuildEC2MXNetGPUInferencePy3DockerImage:
+#    << : *INFERENCE_REPOSITORY
+#    build: &MXNET_GPU_INFERENCE_PY3 false
+#    image_size_baseline: 6758
+#    device_type: &DEVICE_TYPE gpu
+#    python_version: &DOCKER_PYTHON_VERSION py3
+#    tag_python_version: &TAG_PYTHON_VERSION py38
+#    cuda_version: &CUDA_VERSION cu112
+#    os_version: &OS_VERSION ubuntu20.04
+#    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-ec2" ]
+#    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+#    target: ec2
+#    context:
+#      <<: *INFERENCE_CONTEXT
+#  BuildSMMXNetCPUInferencePy3DockerImage:
+#    <<: *INFERENCE_REPOSITORY
+#    build: &MXNET_CPU_INFERENCE_PY3 false
+#    image_size_baseline: 2202
+#    device_type: &DEVICE_TYPE cpu
+#    python_version: &DOCKER_PYTHON_VERSION py3
+#    tag_python_version: &TAG_PYTHON_VERSION py38
+#    os_version: &OS_VERSION ubuntu20.04
+#    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+#    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+#    target: sagemaker
+#    context:
+#      <<: *INFERENCE_CONTEXT
+#  BuildSMMXNetGPUInferencePy3DockerImage:
+#   <<: *INFERENCE_REPOSITORY
+#   build: &MXNET_GPU_INFERENCE_PY3 false
+#   image_size_baseline: 6758
+#   device_type: &DEVICE_TYPE gpu
+#   python_version: &DOCKER_PYTHON_VERSION py3
+#   tag_python_version: &TAG_PYTHON_VERSION py38
+#   cuda_version: &CUDA_VERSION cu112
+#   os_version: &OS_VERSION ubuntu20.04
+#   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+#   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+#   target: sagemaker
+#   context:
+#     <<: *INFERENCE_CONTEXT
+  # BuildMXNetExampleGPUTrainPy3DockerImage:
```

Comment:
fyi: example images are commented out too

Reply:
This should be intentional. @kace to confirm.